src/goGenerateTests.ts

```diff
@@ -112,8 +152,29 @@ function generateTests(conf: Config): Thenable<boolean> {
 				if (err) {
 					return reject('Cannot generate test due to errors: ' + stderr);
 				}
-				let message = 'gotests: ' + conf.msg;
-				vscode.window.showInformationMessage(message);
+
+				let message = stdout;
+				let testsGenerated = false;
+				if (stdout.startsWith(generatedWord)) {
```

Comment:
A comment about the output you expect and what you want to transform it to will make this easier to debug if it changes some day.